ui/app/helpers/clean-keycommand.js

```diff
@@ -0,0 +1,23 @@
+// @ts-check
+import { helper } from '@ember/component/helper';
+
+const KEY_ALIAS_MAP = {
+  ArrowRight: '→',
+  ArrowLeft: '←',
+  ArrowUp: '↑',
+  ArrowDown: '↓',
+  '+': ' + ',
+  // Enter: '⏎',
+  // Tab: '⇥',
+  // Space: '␣',
+  // Shift: '⇧',
+};
+
+export default helper(function cleanKeycommand([part] /*, named*/) {
+  console.log('cleaning', part);
+  Object.keys(KEY_ALIAS_MAP).forEach((key) => {
+    part = part.replace(key, KEY_ALIAS_MAP[key]);
+  });
+
+  return [part];
```

Comment:
`question`:  This returns a list that will be rendered inside of a `span`. If we're returning a list, shouldn't invoke this template helper and assign the return value to a variable and then `each` over that?

Reply:
Wow! So checkout how fail-safe Handlebars is.

![image](https://user-images.githubusercontent.com/41024828/179249946-4d5da13c-6645-47b7-9faf-8f9b9108b554.png)